### PR TITLE
[OpenBLAS] build 64 Aarch without special case

### PR DESCRIPTION
**Pull request by matbesancon**
Following https://github.com/JuliaLang/julia/issues/39436, first attempt at building blas with 64 bits even on aarch64 without the special condition

### CODE REVIEW

**Comment by matbesancon** [comment]
I'm not sure if something special should be made to trigger the various builds here?

File: O/OpenBLAS/common.jl [hunk 1]
```diff
@@ -54,7 +54,7 @@ function openblas_script(;num_64bit_threads::Integer=32, openblas32::Bool=false,
     # Slim the binaries by not shipping static libs
     flags+=(NO_STATIC=1)
 
-    if [[ ${nbits} == 64 ]] && [[ ${target} != aarch64* ]]; then
+    if [[ ${nbits} == 64 ]] then
```
Comment (jmkuhn) [inline O/OpenBLAS/common.jl:57]:
```suggestion
    if [[ ${nbits} == 64 ]]; then
```

**Comment by giordano** [comment]
> I'm not sure if something special should be made to trigger the various builds here?

Yes, you need to touch all the files `O/OpenBLAS/OpenBLAS*@x.y.z/build_tarballs.jl` for the version `x.y.z` you want to rebuild (0.3.13?)

**Comment by ViralBShah** [comment]
By touch, I assume you mean make at least a whitespace change, right? Would be nice to get openblas to 64-bit on aarch64. Will need changes in various other packages as well - and it will be nice to get it done.

**Comment by giordano** [comment]
There is also OpenBLAS high count.

Related to aarch64 OpenBLAS there is https://github.com/JuliaLang/julia/issues/39612, too

**Comment by ViralBShah** [comment]
I think OpenBLAS high count is not crucial at this point (since it cannot be automatically used anyways).

**Comment by matbesancon** [comment]
woops, should not have touched HighCount 3.12?

**Comment by giordano** [comment]
The version is wrong, but it isn't good it's failing....

**Comment by matbesancon** [comment]
also it's failing on x86, so independently from the change of the PR :/

**Comment by giordano** [comment]
Provided that you still need to edit 0.3.13 instead of 0.3.12, I'm very confused by the error: it looks like we're hitting the bug which was supposedly fixed by https://github.com/JuliaPackaging/Yggdrasil/pull/2415 :confused:

**Comment by matbesancon** [comment]
Is there a direction to investigate in particular? Not sure I can be particularly helpful

**Comment by staticfloat** [comment]
@giordano didn't we just add some compiler header fixes for an AVX512 bug?

Googling this error message turns up this: https://gcc.gnu.org/legacy-ml/gcc-patches/2018-01/msg01962.html

**Comment by giordano** [comment]
Yes https://github.com/JuliaPackaging/Yggdrasil/pull/2465#issuecomment-779497053

**Comment by staticfloat** [comment]
Excellent.  In that thread:

> Note this patch doesn't apply to GCC 6 as _mm512_abs_pd and _mm512_mask_abs_pd are not defined

**Comment by giordano** [comment]
But this is failing with libgfortran 4 and 6, so not GCC 6

**Comment by giordano** [comment]
I mean, one possibility is that I built the wrong GCC.....

**Comment by staticfloat** [comment]
I believe this is an OpenBLAS bug.  I can confirm locally that 0.3.12 breaks and 0.3.13 works, so I say we just build 0.3.13 and use that.

**Comment by giordano** [comment]
Good to go?

**Comment by staticfloat** [comment]
Note that this version is going to be incompatible with, like, everything because the aarch64 build is now `ILP64`.

**Pull request merged**